test/src/test/java/lib/form/ValidateButtonTest.java

```diff
@@ -109,13 +109,16 @@ public static final class DescriptorImpl extends Descriptor<TestValidateIsCalled
 
             public void doValidateTest1(@QueryParameter("a") String a, @QueryParameter("b") boolean b,
                                         @QueryParameter("c") boolean c, @QueryParameter("d") String d,
-                                        @QueryParameter("e") String e) {
+                                        @QueryParameter("e") String e,
+                                        @QueryParameter("f") String f
+                                        ) {
```

Comment:
```suggestion
                                        @QueryParameter("e") String e, @QueryParameter("f") String f) {
```

Nitpick for consistency. Feel free to ignore.

(isn't spotless enabled for core?)

Reply:
yes but it's less enabled than in plugins:
https://github.com/jenkinsci/jenkins/blob/b83c92666b5005d4c6e0b01cfda6c409a30bcf5d/pom.xml#L361-L373

vs

https://github.com/jenkinsci/plugin-pom/blob/6949f8af159d30e2722fcf773b397bc870e47359/pom.xml#L665